fail2ban/tests/utils.py

```diff
@@ -208,6 +208,13 @@ def addTest(self, suite):
 	return tests
 
 
+# forwards compatibility of unittest.TestCase for some early python versions
+if not hasattr(unittest.TestCase, 'assertIn'):
+	def __assertIn(self, a, b):
+		return self.assertTrue(a in b)
```

Comment:
I would appreciate if you add a `, msg="%r was not found in %r" % (a, b)"` here so if we use this adapter, we would see what actually caused the failure